lib/class-wp-rest-widget-types-controller.php

```diff
@@ -362,21 +369,21 @@ public function get_item_schema() {
 					'readonly'    => true,
 				),
 				'option_name'                 => array(
-					'description' => __( 'Option name.', 'gutenberg' ),
+					'description' => __( 'DEPRECATED. Option name.', 'gutenberg' ),
```

Comment:
Let's not do this. This data is useful, gutenberg is not the only user of the rest api...

Reply:
Why is it useful?

I don't think clients should need to know about option names as they are an implementation detail of the widget's display and update callbacks.

It's also worth noting that we cannot tell what the option name is for widgets that don't extend `WP_Widget`. For example, consider the following widget. How would the controller know that the option name is `marquee_greeting_text`?

```php
wp_register_sidebar_widget(
	'marquee_greeting',
	'Marquee Greeting',
	function() {
		$greeting = get_option( 'marquee_greeting_text', 'Hello!' );
		printf( '<marquee>%s</marquee>', esc_html( $greeting ) );
	}
);
```

Reply:
Agreed